bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/AbstractTreeViewer.java

```diff
@@ -2466,6 +2525,28 @@ public void setAutoExpandLevel(int level) {
 		expandToLevel = level;
 	}
 
+	/**
+	 * Sets the level of auto expand on Trivial Paths. 0 means that trivial paths
```

Comment:
See above: no auto-expansion should be reflected by `NO_EXPAND` rather than `0`.